vdgsa_backend/rental_viols/models.py

```diff
@@ -1,3 +1,266 @@
+    maker = models.CharField(max_length=50, blank=True, null=True)
+    size = models.CharField(max_length=9, blank=True, null=True)
+    state = models.CharField(max_length=10, blank=True, null=True)
+    value = models.DecimalField(max_digits=6, decimal_places=2, blank=True, null=True)
+    provenance = models.TextField(blank=True, null=True)
+    description = models.TextField(blank=True, null=True)
+    accession_date = models.DateField(blank=True, null=True)
+    notes = models.TextField(blank=True, null=True)
+    viol_num = models.ForeignKey(
+        'Viol', db_column='viol_num',
+        on_delete=models.PROTECT, blank=True, null=True
+    )
+    storer = models.IntegerField(blank=True, null=True)
+
+    program = models.ForeignKey(
+        'Program', db_column='program',
+        on_delete=models.PROTECT, blank=True, null=True
+    )
+
+    def __str__(self) -> str:
+        return (
+            f'{self.bow_num}: {self.maker}'
+        )
+
+    class Meta:
+        managed = True
+
+
+class Case(models.Model):
+    case_num = models.AutoField(primary_key=True)
+    vdgsa_number = models.IntegerField(blank=True, null=True)
+    maker = models.CharField(max_length=50, blank=True, null=True)
+    size = models.CharField(max_length=9, blank=True, null=True)
+    state = models.CharField(max_length=10, blank=True, null=True)
+    value = models.DecimalField(max_digits=6, decimal_places=2, blank=True, null=True)
+    provenance = models.TextField(blank=True, null=True)
+    description = models.TextField(blank=True, null=True)
+    accession_date = models.DateField(blank=True, null=True)
+    notes = models.TextField(blank=True, null=True)
+    viol_num = models.ForeignKey(
+        'Viol', db_column='viol_num',
+        on_delete=models.PROTECT, blank=True, null=True
+    )
+    storer = models.ForeignKey(
+        'Storer', db_column='storer',
+        on_delete=models.PROTECT, blank=True, null=True
+    )
+    program = models.ForeignKey(
+        'Program', db_column='program',
+        on_delete=models.PROTECT, blank=True, null=True
+    )
+
+    def __str__(self) -> str:
+        return (
+            f'{self.case_num} : {self.size} {self.maker} '
+        )
+
+    class Meta:
+        managed = True
+
+
+class Image(models.Model):
+    picture_id = models.AutoField(primary_key=True)
+    vbc_number = models.PositiveIntegerField()
+    type = models.CharField(max_length=4)
+    orig_image_file_name = models.CharField(max_length=250)
+    image_file_name = models.CharField(max_length=250)
+    image_width = models.IntegerField()
+    image_height = models.IntegerField()
+    thumb_file_name = models.CharField(max_length=250)
+    thumb_width = models.IntegerField()
+    thumb_height = models.IntegerField()
+    caption = models.CharField(max_length=3000)
+
+    def __str__(self) -> str:
+        return (
+            f'{self.image_file_name} '
+        )
+
+    class Meta:
+        managed = True
+
+
+class Manager(models.Model):
```

Comment:
I think we can replace this with a custom permission (see the class at the bottom of accounts/models.py)